js/about/contributionStatement.js

```diff
@@ -430,4 +424,243 @@ function longFormattedDateFromTimestamp (timestamp) {
+  },
+  header__titleWrapper: {
+    position: 'relative'
+  },
+  titleWrapper__braveLogo: {
+    backgroundSize: `${braveLogoWidth} ${braveLogoHeight}`,
+    width: braveLogoWidth,
+    height: braveLogoHeight,
+    display: 'inline-block'
+  },
+  titleWrapper__sectionTitle: {
+    color: '#3B3B3B',
+    fontSize: '28px'
+  },
+  sectionTitle__bravePayments: {
+    position: 'relative'
+  },
+  titleWrapper__sectionSubTitle: {
+    color: '#ff5000',
+    fontSize: '15px',
+    position: 'absolute',
+    bottom: '36px',
+    right: '-20px'
+  },
+
+  // ContributionStatementSummaryBox
+  contributionStatement__summaryBox: {
+    marginTop: boxMargin,
+    marginBottom: '25px'
+  },
+  contributionStatement__summaryBoxTable: {
+    borderSpacing: summaryBoxTableBorder,
+    position: 'relative',
+    left: summaryBoxTableBorder
+  },
+  summaryBoxTable__leftColumn: {
+    textAlign: 'right',
+    padding: summaryBoxTablePadding,
+    margin: summaryBoxTableMargin,
+    backgroundColor: lightGray
+  },
+  summaryBoxTable__rightColumn: {
+    textAlign: 'center',
+    padding: summaryBoxTablePadding,
+    margin: summaryBoxTableMargin,
+    border: 'solid 3px #e7e7e7',
+    fontWeight: 'bold'
+  },
+
+  // contributionDateRangeString
+  contributionStatement__dates: {
+    marginBottom: boxMargin
+  },
+
+  // ContributionStatementDetailTable
+  contributionStatementDetailTable__container: {
+    marginTop: boxMargin,
+    marginBottom: boxMargin
+  },
+  contributionStatementDetailTable__table: {
+    width: '100%',
+    border: `5px solid ${lightGray}`
+  },
+  table__siteColumn: {
+    paddingLeft: '40px',
+    textAlign: 'left'
+  },
+  table__headingRow: {
+    background: lightGray
+  },
+  table__spacingRow: {
+    ':before': {
+      lineHeight: '16px',
+      content: "'_'",
+      display: 'block',
+      color: 'white'
+    }
+  },
+  table__th: {
+    whiteSpace: 'nowrap',
+    fontWeight: 'normal'
+  },
+  table__tr: {
+    height: '22px'
+  },
+  table__verified: {
+    left: '-25px'
+  },
+  table__verifiedSite: {
+    position: 'relative',
+    left: '-20px'
+  },
+  verifiedExplainer__wrapper: {
+    marginTop: boxMargin,
+    justifyContent: 'space-between',
+    flexFlow: 'row wrap',
+    paddingLeft: '10px'
+  },
+  verifiedExplainer__text: {
+    marginLeft: '.5em'
+  },
+  verifiedExplainer__pageIndicator: {
+    marginLeft: '.5em'
+  },
+
+  // ContributionStatementFooterNoteBox
+  contributionStatementFooter__noteWrapper: {
+    background: lightGray,
+    padding: containerMargin
+  },
+  noteWrapper__heading: {
+    color: '#ff5000',
+    marginBottom: '5px'
+  },
+
+  // ContributionStatementPageFooter
+  contributionStatementPageFooter__pageFooterWrapper: {
+    margin: `${containerMargin} ${containerMargin} 0`
+  },
+  pageFooterWrapper__body: {
+    color: globalStyles.color.gray
+  },
+
+  // ContributionStatementPage
+  contributionStatement__page: {
+    display: 'flex',
+    flexFlow: 'column nowrap'
```

Comment:
I think you fixed this in next commit which was force pushed over this and the previous one, but `pageBrakeAfter` here (https://github.com/brave/browser-laptop/pull/7887/commits/2f16bbdcd71706a1c29f2fe20183f5835fc93031#diff-dfc3cf3000430cbd1575b0223b06163fR630) should be replaced with `pageBreakAfter`